test/ui/selenium/ViewJobRoles.js

```diff
@@ -0,0 +1,123 @@
+const { resolve } = require("path");
+const { Builder, By } = require('selenium-webdriver');
+const HomePage = require("../POM/HomePage");
+const LoginPage = require('../POM/LoginPage');
+const { get } = require("http");
+const { assert } = require("console");
+const readline = require('readline');
+const email = process.env.LOGIN_EMAIL;
+const password = process.env.LOGIN_PASSWORD;
+const adminEmail = process.env.LOGIN_EMAIL_ADMIN;
+const adminPassword = process.env.LOGIN_PASSWORD_ADMIN;
+require('dotenv').config();
```

Comment:
Is this code the wrong way round? Why is the config being required after already reading from the .env?